packages/cli-platform-apple/src/commands/runCommand/getFallbackSimulator.ts

```diff
@@ -0,0 +1,32 @@
+import {CLIError} from '@react-native-community/cli-tools';
+import {getDestinationSimulator} from '../../tools/getDestinationSimulator';
+import {Device} from '../../types';
+import {FlagsT} from './createRun';
+
+export function getFallbackSimulator(args: FlagsT): Device {
+  /**
+   * If provided simulator does not exist, try simulators in following order
+   * - iPhone 14
+   * - iPhone 13
+   * - iPhone 12
+   * - iPhone 11
+   */
```

Comment:
Not really applicable for all apple platforms

Reply:
Yeah, it's used only on iOS - I don't want to remove it to introduce breaking changes

Reply:
Let's create a new issue to remove this